modular_citadel/code/modules/reagents/chemistry/reagents/healing.dm

```diff
@@ -100,6 +101,7 @@
 	data = list("grown_volume" = 0, "injected_vol" = 0)
 	var/borrowed_health
 	color = "#FFDADA"
+	value = REAGENT_VALUE_VERY_RARE
```

Comment:
"takes a long time" but making it is embarrassingly parallel and cheap--1u of synthflesh becomes 5u of synthtissue